packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.vue

```diff
@@ -11,7 +11,6 @@ const props = withDefaults(defineProps<OnyxRadioButtonProps<TValue>>(), {
   required: false,
   checked: false,
   loading: false,
-  truncation: "ellipsis",
```

Comment:
actually,  I think for the stories/playwright tests, we might even keep it. the group will always overwrite it regardless :)

on top, if the support component is used by a project team directly, they will not have to worry about it 😊

Reply:
I'm checking in the parent component if the truncation prop of the child is undefined and in this case it will always be set and the parent property will be ignored?

Reply:
For now I just added truncation prop for the screenshot tests :)